### ISSUE TITLE
Function ChangeDb()

### ISSUE THREAD

**Issue by jpascal** [Philio/GoMySQL#3]
Function ChangeDb() not return false when database not exist

**Comment by Philio** [comment]
I'll check this out, thanks for the report :)

**Comment by Philio** [comment]
There are a few things with error handling that need cleaning up, an error is still recorded if ChangeDb fails though for the moment:

```
db.ChangeDb("blah")
if db.Errno != 0 {
    fmt.Printf("Error #%d %s\n", db.Errno, db.Error)
    os.Exit(1)
}
```

Output: Error #1049 Unknown database 'blah'

**Comment by Philio** [comment]
This is resolved in 0.2.0 development version, currently in the master branch.
